### ISSUE TITLE
Build training data generator (datagen.py)

### ISSUE THREAD

**Issue by TwoBar** [TwoBar/intl-compiler#6]
Uses ANTHROPIC_API_KEY. 60/30/10 A/B/C split. JSONL to data/<adapter>/. 200-pair validation set. File: intl/datagen.py

**Comment by TwoBar** [comment]
Done in commit 4762d81. All 165 tests passing.
